cgroupspy/interfaces.py

```diff
@@ -24,7 +24,7 @@
 (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 """
-from collections import Iterable
+from collections.abc import Iterable
```

Comment:
```suggestion
try:
    from collections.abc import Iterable
except ImportError:
    from collections import Iterable
```

For Python 2 compatibility.

Reply:
We have no Python 2 support any more.

Reply:
I'd rather add Python > 2 - especially if there will be no other changes

Reply:
Then Python 2 users will be able to use the current version.